src/components/header/loginButton.tsx

```diff
@@ -55,9 +55,22 @@ const LoginButton = () => {
                 >
                   <Popover.Body className="text-center fs-6 px-4">
                     <p>
-                      You need to complete your {user.state === LoginState.NeedsReregistration ? "re-" : ""}
-                      registration with the GHGA Data Portal before you can
-                      start using your LS Login account.
+                      You need to complete your{" "}
+                      {user.state === LoginState.NeedsReregistration
```

Comment:
Can you move this to a function? First, because it is used again below, and second to make the code readable by.